aio/content/navigation.json

```
@@ -369,489 +369,489 @@
+              "tooltip": "Usando Angular con Visual Studio 2015.",
               "hidden": true
             }
           ]
         }
       ]
     },
     {
-      "title": "Configuration",
-      "tooltip": "Workspace and project file structure and configuration.",
+      "title": "Configuración",
+      "tooltip": "Estructura y configuración de archivos de proyecto y espacio de trabajo.",
       "children": [
         {
           "url": "guide/file-structure",
-          "title": "Project File Structure",
-          "tooltip": "How your Angular workspace looks on your filesystem."
+          "title": "Estructura del archivo del proyecto",
+          "tooltip": "Cómo se ve su espacio de trabajo Angular en su sistema de archivos."
         },
         {
           "url": "guide/workspace-config",
-          "title": "Workspace Configuration",
-          "tooltip": "The \"angular.json\" file contains workspace and project configuration defaults for Angular CLI commands."
+          "title": "Configuración del espacio de trabajo",
+          "tooltip": "El archivo \"angular.json\" contiene valores predeterminados de configuración del proyecto y del espacio de trabajo para los comandos CLI de Angular."
         },
         {
           "url": "guide/npm-packages",
-          "title": "npm Dependencies",
-          "tooltip": "Description of npm packages required at development time and at runtime."
+          "title": "Dependencias npm",
+          "tooltip": "Descripción de los paquetes npm necesarios en tiempo de desarrollo y en tiempo de ejecución."
         },
         {
           "url": "guide/typescript-configuration",
-          "title": "TypeScript Configuration",
-          "tooltip": "TypeScript configuration for Angular developers."
+          "title": "Configuración TypeScript",
+          "tooltip": "Configuración de TypeScript para desarrolladores Angular."
         },
         {
           "url": "guide/browser-support",
-          "title": "Browser Support",
-          "tooltip": "Browser support and polyfills guide."
+          "title": "Soporte del navegador",
+          "tooltip": "Soporte de navegador y guía de polyfills."
         },
         {
           "url": "guide/strict-mode",
-          "title": "Strict mode",
-          "tooltip": "Reference documentation for Angular's strict mode."
+          "title": "Modo estricto",
+          "tooltip": "Documentación de referencia para el modo estricto Angular."
         }
       ]
     },
     {
-      "title": "Extending Angular",
-      "tooltip": "Working with libraries and extending the CLI.",
+      "title": "Extendiendo Angular",
+      "tooltip": "Trabajar con bibliotecas y ampliar la CLI.",
       "children": [
         {
-          "title": "Angular Libraries",
-          "tooltip": "Extending Angular with shared libraries.",
+          "title": "Bibliotecas Angular ",
+          "tooltip": "Ampliación de Angular con bibliotecas compartidas.",
```

Segun el issue https://github.com/angular-hispano/angular/issues/9, vamos a usar librería en vez de biblioteca
```suggestion
          "tooltip": "Ampliación de Angular con librerías compartidas.",
```